lib/Dialect/Torch/Transforms/RefineTypes.cpp

```diff
@@ -664,7 +664,7 @@ void TypeAnalysis::visitOperation(Operation *op,
           AtenUnsqueezeOp, AtenViewOp, Aten_UnsafeViewOp, AtenReshapeOp,
           Aten_ReshapeAliasOp, AtenResize_Op, AtenTransposeIntOp, AtenTOp,
           AtenPermuteOp, AtenIndexSelectOp, AtenSelectIntOp,
-          AtenSelectScatterOp, AtenNarrowOp, AtenSliceTensorOp,
+          AtenSelectScatterOp, AtenNarrowOp, AtenSliceTensorOp,AtenScatterReduceTwoOp,
```

Comment:
Do you still need to add this here?

Reply:
For now yes, I will have to add this to the dtype branch at some point as well